test/clj/game_test/cards/hardware.clj

```diff
@@ -1177,3 +1177,19 @@
    (prompt-choice :runner "Expose a card")
    (prompt-select :runner (get-ice state :archives 0))
    (is (= 3 (:credit (get-runner))) "Gained 1 more credit from exposing")))
+
+(deftest zer0
+  ;; Zer0 - Once per turn, deal 1 damage to self, to gain 1 credit and 2 cards.
+  (do-game
+    (new-game (default-corp)
+              (default-runner [(qty "Zer0" 1) (qty "Sure Gamble" 2) (qty "Corroder" 1)]))
```

Comment:
The single `qty` cards could be replaced with just a string of the card name (e.g. `"Zer0"`)